controllers/csm_controller.go

```diff
@@ -465,7 +465,7 @@ func (r *ContainerStorageModuleReconciler) handleDaemonsetUpdate(oldObj interfac
 // ContentWatch - watch updates on deployment and deamonset
 func (r *ContainerStorageModuleReconciler) ContentWatch() error {
 
-	sharedInformerFactory := sinformer.NewSharedInformerFactory(r.K8sClient, time.Duration(time.Hour))
+	sharedInformerFactory := sinformer.NewSharedInformerFactory(r.K8sClient, time.Duration(4*time.Minute))
```

Comment:
@kumarkgosa, instead of changing the duration of reconcile time, could we check why Powerflex driver takes lot of time to be up and running. Without this change, Powerscale and Powerstore drivers are up and running. Changing the reconcile to 4 mins may not be the right solution here. Triggering reconcile for each of the driver every 4 minutes may not be a good idea